packages/tfchain_client/src/tftBridge.ts

```diff
@@ -29,5 +37,30 @@ class QueryTFTBridge {
     );
     return eventData[0].amount.toPrimitive();
   }
+
+  @checkConnection
+  async GetWithdrawFee(): Promise<string> {
+    const fee = await this.client.api.query.tftBridgeModule.withdrawFee();
+    return fee.toString();
```

Comment:
we can use here `toPrimitive` instead of `toString` to not force it to be a string in case it's a number